packages/nimble-components/src/patterns/button/types.ts

```diff
@@ -32,3 +42,14 @@ export const ButtonAppearance = {
 } as const;
 export type ButtonAppearance =
     typeof ButtonAppearance[keyof typeof ButtonAppearance];
+
+/**
+ * Types of button appearance variants.
+ * @public
+ */
+export const ButtonAppearanceVariant = {
```

Comment:
Don't forget to add a test for this type, following the pattern of the existing type.